### PR TITLE
Issue1347 missing factus metric docs

### PR DESCRIPTION
**Pull request by samba2**
adds documentation about the metrics currently exposed by Factus

### CODE REVIEW

**Comment by samba2** [comment]
I ran a manual check using the `serve.sh` script and the cross linking looked fine

File: factcast-site/documentation/content/factus/metrics.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+weight = 1050
+
++++
+
+Like [the Factcast server]({{< ref "/setup/server/metrics.md" >}} "factcast server metrics"), also Factus makes use 
+of [micrometer.io](https://micrometer.io/) metrics.
+
+### Metric namespaces and their organization
+
+At the time of writing, there are four namespaces exposed:
```
Comment (uweschaefer) [inline factcast-site/documentation/content/factus/metrics.md:22]: four or three?

File: factcast-site/documentation/content/factus/metrics.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
++++
+draft = false
+title = "Metrics"
+description = ""
+
+
+creatordisplayname = "Uwe Schaefer"
```
Comment (uweschaefer) [inline factcast-site/documentation/content/factus/metrics.md:7]: this should be you?

**Comment by uweschaefer** [comment]
lgtm

**Pull request merged**